docs/notebooks/Notebook_01_buildH_calc_OP.ipynb

```diff
@@ -403,7 +403,7 @@
     "\n",
     "$$OP_{stem}(CH) = \\frac{ \\frac{1}{N} \\sum_{i=1}^{N} OP(CH)_i }{\\sqrt{N}}$$\n",
     "\n",
-    "where *N* is the number of lipids, $OP(CH)_i$ is the mean order parameter of a given CH of residue $i$ (this $OP(CH)_i$ is itself an average of the order parameter of a given CH of residue *i* over frames). In fact, we first average over frames, then we average over residues to compute the STEM.\n",
+    "where *N* is the number of lipids, $OP(CH)_i$ is the mean order parameter of a given CH of residue $i$ (this $OP(CH)_i$ is itself an average of the order parameter of a given CH of residue *i* over frames). In fact, we first average over frames, then we average over residues to compute the STEM. More information about the order parameter can be found in the documentation: [Order parameters and statistics](../order_parameter.md)\n",
```

Comment:
Thinking about that, I realize I don't use the same indices in this formula and the one in "OP & statistics". I need to change this!

Reply:
Okay, I let you correct this once this is merged